charts/rekor/README.md

```diff
@@ -169,6 +169,7 @@ Part of the sigstore project, Rekor is a timestamping server and transparency lo
 | server.sharding.filename | string | `"sharding-config.yaml"` |  |
 | server.sharding.mountPath | string | `"/sharding"` |  |
 | server.signer | string | `"memory"` |  |
+| server.signerPassword | string | `""` |  |
```

Comment:
should we add a disclaimer this should only be used for testing?
passing password string here seems like bad practice in general

Reply:
Another option is to omit this entirely, as extraArgs can cover this. WDYT?

Reply:
whats the motivation/use case for adding this arg?

extraArgs is also an option, especially it seems tedious to update the helm chart for every new flag

Reply:
Just had someone accessing about signing options for Rekor and I noticed this was missing. Have we formed guidance on when we should add a top-level argument vs using extraArgs?

Reply:
Thinking this over, I'm gonna close this out.